draft-ietf-quic-transport.md

```diff
@@ -2432,17 +2432,18 @@ The capacity available on the new path might not be the same as the old path.
 Packets sent on the old path MUST NOT contribute to congestion control or RTT
 estimation for the new path.
 
-On confirming a peer's ownership of its new address, an endpoint MUST
+On confirming a peer's ownership of its new address, an endpoint SHOULD
 immediately reset the congestion controller and round-trip time estimator for
 the new path to initial values (see Appendices A.3 and B.3 in {{QUIC-RECOVERY}})
-unless it has knowledge that a previous send rate or round-trip time estimate is
-valid for the new path.  For instance, an endpoint might infer that a change in
-only the client's port number is indicative of a NAT rebinding, meaning that the
-new path is likely to have similar bandwidth and round-trip time. However, this
-determination will be imperfect.  If the determination is incorrect, the
-congestion controller and the RTT estimator are expected to adapt to the new
-path.  Generally, implementations are advised to be cautious when using previous
-values on a new path.
+unless the only change is the other endpoint's port number.  Because
+port-only changes are commonly the result of NAT rebinding or other middlebox
+activity, the endpoint MAY instead retain its congestion control state and
+round-trip estimate in those cases instead of reverting to initial values.
+In cases where congestion control state
+retained from an old path is used on a new path with substantially different
+characteristics, a sender may transmit too aggressively until the congestion
+controller and the RTT estimator have adapted. Generally, implementations are
+advised to be cautious when using previous values on a new path.
```

Comment:
This last sentence sounds like it generally good advise to reuse (just be cautious about it).  I don't think that is a general good advise for all kind of value. For some it might be (with cautious) but in any case I would need more words that this in order to give any recommendation.

Reply:
Sorry I know this sentence was there in the old text but given we ware looking at this we should either remove this part of give more advise/pointers.

Reply:
I read that sentence completely differently, but maybe "implementations should be careful when using" is clearer?

Reply:
This is not a recommendation. The advise there seems appropriate to me; is there specific other advise that is needed?